memory/dma_memtest.py

```diff
@@ -49,6 +49,7 @@ def setUp(self):
         copies of the linux kernel that will be uncompressed.
         """
         self.nfail = 0
+        self.base_dir = None
```

Comment:
why is this initialised to None ?
It can be `self.base_dir = os.path.join(self.tmpdir, 'linux.orig')` here, right ?

Reply:
Right. Will move it after initializing `self.tmpdir`.